lms/templates/word_cloud.html

```diff
@@ -1,30 +1,45 @@
+<%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 
-<section
+<div
     id="word_cloud_${element_id}"
     class="${element_class}"
     data-ajax-url="${ajax_url}"
 >
+    % if display_name is not display_name_default:
+    <h3 class="hd hd-3">${display_name}</h3>
+    % endif
+
+    % if instructions is not instructions_default:
+    <div class="input_cloud_section" role="group" aria-labelledby="word_cloud_${element_id}_instructions">
+        <div class="input_cloud_instructions" id="word_cloud_${element_id}_instructions">
+            ${instructions}
+        </div>
+    % else:
+    <div class="input_cloud_section" role="group">
+    % endif
 
-    <section class="input_cloud_section">
         % for row in range(num_inputs):
+            <label for="word_cloud_${element_id}_input_${row}" class="sr">${row + 1}/${num_inputs}</label>
             <input
+                id="word_cloud_${element_id}_input_${row}"
                 class="input-cloud"
                 ${'style="display: none;"' if submitted else ''}
                 type="text"
                 size="40"
             />
         % endfor
 
-        <section class="action">
+        <div class="action">
             <input class="save" type="button" value="${_('Save')}" />
-        </section>
-    </section>
+        </div>
+    </div>
 
-    <section id="result_cloud_section_${element_id}" class="result_cloud_section">
-        <h3>${_('Your words:')} <span class="your_words"></span></h3>
-        <h3>${_('Total number of words:')} <span class="total_num_words"></span></h3>
+    <div id="result_cloud_section_${element_id}" class="result_cloud_section">
         <div class="word_cloud"></div>
-    </section>
+        <p><strong><span class="total_num_words"></span></strong> ${'words submitted in total'}.</p>
+        <p>${'Your words were'}:</p>
```

Comment:
This needs to be translated, and include the colon in the translated string, as it won't make sense to be concatenated in all languages.